app/assets/stylesheets/log.scss

```diff
@@ -0,0 +1,7 @@
+ul.dispatch-log{
+  li{
+    .edit-form{
+      display: none;
+    }
+  }
+}
```

Comment:
Could've written as `ul.dispatch-log li .edit-form`, but I anticipate more styling will be added to this file.